src/lib/MnemonicPhrase.js

```diff
@@ -0,0 +1,174 @@
+class MnemonicPhrase {
```

Comment:
Note to other reviewers: This file was already approved in PR https://github.com/nimiq/keyguard-next/pull/17.